CONTRIBUTING.md

```diff
@@ -158,3 +223,319 @@ including how to setup a local Kubernetes cluster, can be found in
+  closure.
+* for issues, label the estimated size (if known). The size can be updated
+  during backlog grooming by the contributors. This estimate is used to guide
+  sprint scoping the exercise.
+
+### Issue Triage
+
+Once an issue has been submitted, the CI (GitHub actions) or a human will
+automatically review the submitted issue or PR to ensure that it has all relevant
+information. If information is lacking or there is another problem with the
+submitted issue, an appropriate [`triage/<?>`](#triage) label will be
+applied.
+
+After an issue has been triaged, the maintainers can prioritize the issue with
+an appropriate [`priority/<?>`](#priority) label.
+
+Once an issue has been submitted, categorized, triaged, and prioritized it
+is marked as `ready-to-work`. A ready-to-work issue should have labels
+indicating assigned areas, prioritization, and should not have any remaining
+triage labels.
+
+### Issue Kinds
+
+Use an kind label to describe the kind of issue you are submitting. Valid
+kinds include:
+
+* [`kind/api-change`](#api-change) -- for api changes
+* [`kind/bug`](#bug) -- for filing a bug
+* [`kind/cleanup`](#cleanup) -- for code cleanup and organization
+* [`kind/deprecation`](#deprecation) -- for deprecating a feature
+* [`kind/design`](#design) -- for proposing a design or architectural change
+* [`kind/documentation`](#documentation) -- for updating documentation
+* [`kind/failing-test`](#failing-test) -- for reporting a failed test (may
+  create with automation in future)
+* [`kind/feature`](#feature) -- for proposing a feature
+* [`kind/support`](#support) -- to request support (please try using our
+  [Slack](https://kubernetes.slack.com/archives/CR2J23M0X) channel for
+  interactive help)
+
+#### API Change
+
+A `kind/api-change` label categorizes an issue or PR as related to adding, removing,
+or otherwise changing an API.
+
+All API changes must be reviewed in addition to the standard code review and
+approval workflow. MAINTAINERS and code OWNERS can approve an API change by
+commenting with `/approve-api-change`.
+
+To create an API change issue:
+
+* label your issue with `kind/api-change`
+* describe in the issue body which API you are changing making sure to include
+  * API endpoint and schema (endpoint, Version, APIGroup, etc.)
+  * Is this a breaking change?
+  * Can new or older clients opt-in to this API?
+  * Is there a fallback? What are the implications of not supporting this API version?
+
+Before starting any work on an API change it is important that you have proper
+review and approval from the project maintainers.
+
+#### Bug
+
+A `kind/bug` label categorizes an issue or PR as related to a bug.
+
+Any problem encountered when building, configuring, or running Antrea could be a
+potential case for submitting a bug.
+
+To create a bug issue:
+
+* label your issue with `kind/bug`
+* describe your bug in the issue body making sure to include:
```

Comment:
more information may be worth mentioning (e.g. container runtime) - see current bug template